lib/match-conditions.js

```diff
@@ -3,8 +3,12 @@
 var fs = require('fs');
 var execSync = require('child_process').execSync;
 var _ = require('lodash');
+var semver = require('semver');
 
 var version = process.version;
+var semVersion = semver.major(process.version) + '.' +
```

Comment:
This part is for removing of `-pre` from node version (e.g. `v8.0.0-pre`). Semver does not work as expected if it is included in the version, and I didn't found any 'nicer' way of doing that.